Functions.Templates/ProjectTemplate/CSharp-Isolated/Company.FunctionApp.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
hmm I don't know if this needs to change necessarily.
For v4 Functions, are we thinking that the default for .NET Isolated is .NET 6.0? If so, then this makes sense.